shared/chat/conversation/attachment-get-titles/index.tsx

```diff
@@ -96,89 +83,174 @@ class GetTitles extends React.Component<Props, State> {
     if (!info) return null
 
     return (
-      <Kb.ScrollView style={{height: '100%', width: '100%'}}>
-        <Kb.Box style={Styles.isMobile ? stylesMobile : stylesDesktop}>
-          <Kb.Box style={{...Styles.globalStyles.flexBoxCenter, height: 150, width: 150}}>
-            {info.type === 'image' ? (
-              <Kb.OrientedImage
-                src={Styles.isAndroid ? `file://${path}` : path}
-                style={Styles.isMobile ? {height: 150, width: 150} : {maxHeight: '100%', maxWidth: '100%'}}
+      <Kb.ScrollView style={styles.scrollView}>
+        <Kb.Box style={styles.container}>
+          <Kb.Box2
+            alignItems="center"
+            direction="vertical"
+            fullWidth={true}
+            fullHeight={Styles.isMobile}
+            style={styles.upperContainer}
+          >
+            <Kb.Box style={styles.imageContainer}>
+              {info.type === 'image' ? (
+                <Kb.OrientedImage src={Styles.isAndroid ? `file://${path}` : path} style={styles.image} />
+              ) : (
+                <Kb.Icon type="icon-file-uploading-48" />
+              )}
+            </Kb.Box>
+            {paths.length > 0 && !Styles.isMobile && (
+              <Kb.Box2 direction="vertical" style={styles.filename}>
+                <Kb.Text type="BodySmallSemibold">Filename</Kb.Text>
+                <Kb.Text type="BodySmall">
+                  {info.filename} ({this.state.index + 1} of {paths.length})
+                </Kb.Text>
+              </Kb.Box2>
+            )}
+            <Kb.PlainInput
+              style={styles.input}
+              autoFocus={true}
+              autoCorrect={true}
+              placeholder={titleHint}
+              multiline={true}
+              value={info.title}
+              onEnterKeyDown={this._onNext}
+              onChangeText={this._updateTitle}
+              selectTextOnFocus={true}
+            />
+          </Kb.Box2>
+          <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.buttonContainer}>
```

Comment:
I think this should be `Kb.ButtonBar`

Reply:
It was, but I kept running into issues with it and just went with `Box2` instead. I'll try `ButtonBar` again, though.